app/src/main/kotlin/org/bspb/smartbirds/pro/sync/UploadManager.kt

```diff
@@ -53,11 +54,9 @@ open class UploadManager {
     @Bean
     protected lateinit var monitoringManager: MonitoringManager
 
-    @StringRes(R.string.tag_lat)
-    protected lateinit var tagLatitude: String
+    protected var tagLatitude: String = "latitude"
 
-    @StringRes(R.string.tag_lon)
-    protected lateinit var tagLongitude: String
+    protected var tagLongitude: String = "longitude"
```

Comment:
Seems the change is out of scope of the PR

Reply:
https://trello.com/c/N6q7mARn/174-fix-coordinates-null-check-in-uploadmanager